src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -300,4 +353,28 @@ private void fetchIssue() throws IOException {
             fetchedIssueDetails = true;
         }
     }
+
+    private static class DraftReviewComment {
```

Comment:
It's just my 2 cents opinion but why don't you make this public and add the GH prefix on it too to make it standard with the majority of the other classes in the project.